READMEs/c2d-flow-more-examples.md

```diff
@@ -76,7 +76,7 @@ ALGO_url_file = UrlFile(
 
 Install dependencies:
 
-`pip install Pillow`
+`pip3 install Pillow`
```

Comment:
please revert changes referencing pip3 and python3. If the venv is activated, the correct version of python will be used.